packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -45,19 +59,54 @@ export function storeActivityTypes(activityTypes) {
   };
 }
 
-export const FETCHING_SPACE_DETAILS = `widget-space/FETCHING_SPACE_DETAILS`;
-export function fetchingSpaceDetails() {
+
+function fetchingSpaceDetails() {
   return {
     type: FETCHING_SPACE_DETAILS
   };
 }
 
+/**
+ * Gets details about the space
+ * @param {Object} sparkInstance
+ * @param {String} spaceId
+ * @returns {Thunk}
+ */
 export function getSpaceDetails(sparkInstance, spaceId) {
   return (dispatch) => {
+    let spaceIdUUID;
+    if (isUuid(spaceId)) {
+      spaceIdUUID = spaceId;
+    }
+    else {
+      let isEncodedUUID = false;
+      let base64uuid;
+      // Check for base 64 encoding
+      try {
+        base64uuid = atob(spaceId).split(`/`).pop();
```

Comment:
May want to put this into the utils package since we use it a lot. Also the other direction as well